### ISSUE TITLE
Feature request - small bar setting

### ISSUE THREAD

**Issue by senao-mis** [a85/POSTMan-Chrome-Extension#1286]
![2015-07-24 10 47 40](https://cloud.githubusercontent.com/assets/4914276/8866672/a1c663e8-31f1-11e5-8c47-ed8b12618cab.png)
The above image is my 15" screen for postman 3.0
the response region is just too small, owing to the setting bars region takes too much space :p

Any possible to narrow it?

**Comment by senao-mis** [comment]
My bad, I just noticed that I can scroll the header area while mouse on it..
This is much better, but still any possible to change the line-height?

![2015-07-24 10 56 00](https://cloud.githubusercontent.com/assets/4914276/8866752/9acd4cb8-31f2-11e5-846f-21437c2f03de.png)

**Comment by abhijitkane** [comment]
You can change the response font size by going to Settings > Response Font Size. The Settings icon is located in the top-right corner.

**Comment by senao-mis** [comment]
I just tried , it is not exactly what I mean, request header owns too much area.
![Postman 2 response area V.S. Postman 3 response area](https://cloud.githubusercontent.com/assets/4914276/8872453/b42d655c-322f-11e5-8404-8aeda7d73cff.png)
But I admit this feature depended on personal preference, and the current layout is not too bad to use.

And I found the "UI size" seems not working well, the default one owns smallest UI font size, but the UI elements line-height are same as small, smallest.
I will make another ticket for this.

Below are screenshots with UI size setting in "bigger", "default", "smallest".
![big-size](https://cloud.githubusercontent.com/assets/4914276/8872350/becbd0bc-322e-11e5-8341-8ae0a4b121af.png)

![default-size](https://cloud.githubusercontent.com/assets/4914276/8872376/fbf46e54-322e-11e5-8268-cee9e80d82d5.png)

![samllest](https://cloud.githubusercontent.com/assets/4914276/8872378/02ba9632-322f-11e5-91a3-ab85fb47443c.png)
